oaknational/Oak-Web-Application
# New curriculum MV behind flag

## Description
This enables the new curriculum MV behind a `const`ant. By enabling `ENABLE_NEW_CURRIC_MV` in `src/utils/curriculum/constants.ts`

## Issue(s)

Fixes `CUR-1002`

## How to test
`ENABLE_NEW_CURRIC_MV = false` nothing should have changed (the default)

1. Go to https://deploy-preview-2954--oak-web-application.netlify.thenational.academy
2. Test for regressions

`ENABLE_NEW_CURRIC_MV = true` enabled the visualiser should be using the new MV

1. Go to https://deploy-preview-2954--oak-web-application.netlify.thenational.academy
2. Check we're using the new MV, you can use units in [Potential incorrect curriculum sequences](https://www.notion.so/oaknationalacademy/Potential-incorrect-curriculum-sequences-13c26cc4e1b180839306ee2181ec3e4f) to verify
